nixos/doc/manual/redirects.json

```diff
@@ -1070,18 +1070,6 @@
   "module-services-taskserver-manual-ca-management": [
     "index.html#module-services-taskserver-manual-ca-management"
   ],
-  "module-services-sourcehut": [
-    "index.html#module-services-sourcehut"
```

Comment:
Please don't simply delete the references but redirect them to an anchor in the release notes. Otherwise there's no way for people who aren't on GitHub follow some existing link to this documentation (e.g. from a forum or bookmark) to figure out what's happened.